react-native/android/src/main/jni/Application.mk

```diff
@@ -1,7 +1,7 @@
 APP_BUILD_SCRIPT := Android.mk
 
 APP_ABI := armeabi-v7a x86 x86_64 arm64-v8a
-APP_PLATFORM := android-9
+APP_PLATFORM := android-21
```

Comment:
I am a bit concerned with this bump. RN is currently at 16 (https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/jni/Application.mk#L9). On the other hand, 21 is Lollipop and we will support roughly 95 % of all active devices.

Reply:
<img width="1363" alt="Screenshot 2020-10-09 at 10 08 31" src="https://user-images.githubusercontent.com/3899337/95559085-7e095b00-0a17-11eb-8fbc-9a6dd913adc7.png">

Cool, I've tested with ```Android-16``` and seems to works will move it to that. 👍

Reply:
yes RN 0.59 should be able to run on android 9, so we should not be bumping this version.

Reply:
Cool, then we need to change the documentation on the Readme.md to the correct NDK version or provide some workaround.